src/appleseed/renderer/modeling/frame/frame.cpp

```diff
@@ -720,27 +763,34 @@ bool Frame::write_aov_images(const char* file_path) const
 
     bool success = true;
 
-    if (!aovs().empty())
```

Comment:
I do not check for the size anymore because if I do, I would have to check `aovs().empty()` and `aov_images().empty()`. Otherwise extra AOVs wouldn't be saved if there isn't any main AOV.

Reply:
Maybe I'm wrong, there is always an AOV, beauty, right ? @est77

Reply:
beauty is not an AOV in appleseed. There's an image in the frame to store it.